akka-docs/src/main/paradox/stream/operators/Source-or-Flow/grouped.md

```diff
@@ -17,6 +17,17 @@ Accumulate incoming events until the specified number of elements have been accu
 Accumulate incoming events until the specified number of elements have been accumulated and then pass the collection of
 elements downstream.
 
+## Examples
```

Comment:
Did we decide on having the examples above the RS semantics?

I don't remember and we're not doing it consistently now - happy to help make it consistent if this is what we decided on ;)

Reply:
I think we did decide on that

Reply:
:+1: I'll PR that then

Reply:
https://github.com/akka/akka/pull/28357